code/modules/spells/spell_types/lichdom.dm

```diff
@@ -87,6 +87,9 @@
 					var/mob/living/carbon/C = old_body
 					for(var/obj/item/W in C)
 						C.unEquip(W)
+					for(var/obj/item/organ/internal/I in C.internal_organs)
```

Comment:
Before I get any comments on this, things placed into people during operations (Nuke disks, clown masks, bombs, and so on) go into this list. They probably shouldn't, but they do, so I can't do my happy istypeless for loops here ;_;